elastalert/elastalert.py

```diff
@@ -1129,6 +1133,14 @@ def run_all_rules(self):
                 elastalert_logger.info("Ran %s from %s to %s: %s query hits (%s already seen), %s matches,"
                                        " %s alerts sent" % (rule['name'], old_starttime, pretty_ts(endtime, rule.get('use_local_time')),
                                                             total_hits, self.num_dupes, num_matches, self.alerts_sent))
+
```

Comment:
let's extract this to a function, will be easier to merge with their changes that way until they merge the PR

Reply:
done

Reply:
i meant the block, not the call to the statsd client :)

Reply:
done